charts/telemetry/values.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+telemetry:
+
+  schedule: '"0 * * * *"'
+
+  kubermaticAgent:
+    image:
+      repository: quay.io/kubermatic/telemetry-kubermatic-agent
+      pullPolicy: IfNotPresent
+      tag: "v0.1.0"
+
+  kubernetesAgent:
+    image:
+      repository: quay.io/kubermatic/telemetry-kubernetes-agent
+      pullPolicy: IfNotPresent
+      tag: "v0.1.0"
+
+  reporter:
+    image:
+      repository: quay.io/kubermatic/telemetry-reporter
+      pullPolicy: IfNotPresent
+      tag: "v0.1.0"
+
+  serviceAccount:
+    name: telemetry-agent-sa
+    namespace: telemetry-system
+
+  uuidgen: ""
```

Comment:
Please the uuid function from helm: https://helm.sh/docs/chart_template_guide/function_list/#uuid-functions
not passing from here

Reply:
Wouldn't this create a new UUID every time the chart is installed/updated?

I also would not call it "uuidgen", but "uuid" if anything. "uuidgen" is a program on my computer.

Reply:
Oh, yeah, you are right, this will generate new uuid every time.
Then let's pass the uuid from here to the chart.
and I agree that we should use `uuid` not `uuidgen`.